lang/cs/Org.Apache.REEF.Common/Parameters/DriverConfigurationProviders.cs

```diff
@@ -21,10 +21,10 @@
 using Org.Apache.REEF.Tang.Annotations;
 using Org.Apache.REEF.Tang.Interface;
 
-namespace Org.Apache.REEF.Client.API.Parameters
+namespace Org.Apache.REEF.Common.Parameters
```

Comment:
The namespace should still reflect the fact that this is a parameter on the client side. Maybe `Common.Client.Parameters`?